File: workflows/pythonSQL2in2out/code/src/main/python/graph/Reformat0.py
from pyspark.sql import SparkSession
from pyspark.sql.types import *
from pyspark.sql.functions import *
from ..config import Config
from ..prophecylibs import *

@Visual(id = "Reformat0", label = "Reformat0", x = 515, y = 82, phase = 0)
def Reformat0(spark: SparkSession, _in: DataFrame) -> Reformat:
    out = _in.select()

    return out

File: workflows/TestCoalesce/code/src/main/scala/graph/Reformat0.scala
package graph

import org.apache.spark.sql.types._
import io.prophecy.libs._
import io.prophecy.libs.UDFUtils._
import io.prophecy.libs.Component._
import io.prophecy.libs.DataHelpers._
import io.prophecy.libs.SparkFunctions._
import io.prophecy.libs.FixedFileFormatImplicits._
import org.apache.spark.sql.ProphecyDataFrame._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import config.ConfigStore._
import udfs.UDFs._
import udfs._
import graph._

@Visual(id = "Reformat0", label = "Reformat0", x = 672, y = 107, phase = 1)
object Reformat0 {

  def apply(spark: SparkSession, in: DataFrame): Reformat = {
    import spark.implicits._

    val out = in.select(
      col("customer_id"),
      col("phone"),
      col("account_open_date")
    )

    out

  }

}
